modules/ROOT/pages/partner-manager-identifiers.adoc

```diff
@@ -1,3 +1,90 @@
 = Identifiers
 
-Identifiers are used to identify the sender and receiver partners that are involved in a B2B transaction. For X12 messages, the identifiers in the ISA and GS headers provide valuable information that enables identification of the appropriate flow to use to process the transaction.
\ No newline at end of file
+Identifiers are used to identify the sender and receiver partners that are involved in a B2B transaction. For X12 messages, the identifiers in the ISA and GS headers provide valuable information that enables identification of the appropriate flow to use to process the transaction.
```

Comment:
Can we reword "are used" to make the sentence active? Who or what uses the identifiers to do this?

Reply:
Can this intro text refer somehow to the topic's subtopics, to set the audience expectation of WHAT ABOUT identifiers is contained within?

It might be a simple way to let the audience know that although the task comes first, it is followed by reference information about the type values and other factors that are named in the UI and manipulated by the user in the task.